src/main/java/org/javarosa/xpath/expr/XPathEncryptStringFunc.java

```diff
@@ -0,0 +1,85 @@
+package org.javarosa.xpath.expr;
+
+
+import org.commcare.util.Base64;
+import org.commcare.util.Base64DecoderException;
+import org.javarosa.core.model.condition.EvaluationContext;
+import org.javarosa.core.model.instance.DataInstance;
+import org.javarosa.xpath.XPathException;
+import org.javarosa.xpath.parser.XPathSyntaxException;
+
+import javax.crypto.Cipher;
+import javax.crypto.SecretKey;
+import javax.crypto.spec.SecretKeySpec;
+import java.nio.ByteBuffer;
+import java.nio.charset.Charset;
+import java.security.SecureRandom;
+
+public class XPathEncryptStringFunc extends XPathFuncExpr {
+    public static final String NAME = "encrypt";
```

Comment:
I think `encrypt-string` was a stronger name for this method and reduces ambiguity, so I'd propose changing this back (easy enough to change the test form function name)

Reply:
OK, I've changed it back to encrypt-string.